exercise_01.py

```diff
@@ -0,0 +1,15 @@
+def displayPattern(n):
+    n_to_print = ""
+    total = ""
+    for i in range(1, n + 1):
+        n_to_print = "{} {}".format(i, n_to_print)
```

Comment:
Why aren't you using f-strings 🤦?

Reply:
I don't understand why you are not seeing this comment... is that palm face emoji distracting you from what I am asking you?